examples/src/main/proto/testservice.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```

Comment:
Why was this file added? It seems unused.

Reply:
Oops removed